test/integration-server/acrolinx-endpoint.test.ts

```diff
@@ -482,6 +482,28 @@ describe('e2e - AcrolinxEndpoint', () => {
       });
     });
 
+    //Acrolinx Live target needs to be assigned for the following test case 
+    describe('liveSearch', () => {
+      it.skip('should get suggestions for a live search', async () => {
+        const liveRequest = {
+          'request-id': 'abc1-Request',
+          phrase: 'phrase from the document',
+          target: '79ec4ac3-a9e3-4532-9d01-ae6141aefea2'
+        };
+        const liveSearchResult = await api.getLiveSuggestions(ACROLINX_API_TOKEN, liveRequest);
+        expect(liveSearchResult.requestId).toEqual(liveRequest['request-id']);
+        expect(liveSearchResult.results).toBeTruthy();
+      });
+    });
+
+    describe('Checking capabilities for Acrolinx Live supported targets', () => {
+      it.skip('Checking capabilities should have an additional acrolinxLive flag', async () => {
```

Comment:
why skip the test?

Reply:
This is a new feature on the backend currently only on https://unstable.tds.cld.ax/ and should be released in 2022.12. Until then, the test will fail.